src/components/Input/__tests__/Input.spec.tsx

```diff
@@ -89,16 +92,100 @@ describe('<Input> that renders an input field', () => {
     it('should call change callback correctly', async () => {
         const user = userEvent.setup();
         const onChange = jest.fn();
-        view = renderComponent({ onChange });
+        view = renderInputComponent({ onChange });
 
         await user.type(screen.getByRole('textbox'), 'test');
 
         expect(onChange).toHaveBeenCalled();
     });
 
     it('should add string html attributes correctly', () => {
-        view = renderComponent({ 'data-test': 'something' });
+        view = renderInputComponent({ 'data-test': 'something' });
 
         expect(screen.getByRole('textbox')).toHaveAttribute('data-test', 'something');
     });
 });
+
+describe('<SearchInput> that renders an input field', () => {
+    let view: RenderResult;
```

Comment:
since there is no beforeEach rendering the result, it may be better to have a local variable inside of each test, not sure if with test parallelisation it would conflict somehow.